dotcom-rendering/src/lib/useHydrated.ts

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react';
+import { useOnce } from './useOnce';
+
+export const useHydrated = (): boolean => {
+	const [hydrated, setHydrated] = useState(false);
+	useOnce(() => {
```

Comment:
Think the `useOnce` is unnecessary here as its use case is to check all of the items in the dependency array to be defined before invoking the callback. In this case there is no dependency array so a regular useEffect with an empty dependency array should ensure the useEffect is called once.